tests/yast2_gui/yast2_hostnames.pm

```diff
@@ -17,37 +17,47 @@
 use base "y2x11test";
 use strict;
 use testapi;
-use utils qw(type_string_slow clear_console);
+use utils 'clear_console';
+use x11utils qw(turn_off_kde_screensaver turn_off_gnome_screensaver);
 
 sub run {
-    my $self   = shift;
-    my $module = "host";
-
+    my $self         = shift;
+    my $module       = "host";
+    my $dm           = lc get_var('DESKTOP');
```

Comment:
care to use `get_required_var` instead?

Reply:
oh, I have forgotten about this subroutine. Ty!